### DISCUSSION TITLE
When can we expect 1.1.63 release (that includes update to ch.qos.logback:logback-classic from 1.2.10 to 1.3.12) ?

### DISCUSSION THREAD

**Discussion by ramaprasadcr** [red6/pdfcompare#138] [General]
We have a vulnerability issue where package ch.qos.logback:logback-classic should be updated from 1.2.10 to 1.3.12. I see the branch already updated. When can we expect a new release of the library?
Thank you very much

**Comment by finsterwalder** [comment]
Was already released yesterday.

**Comment by finsterwalder** [comment]
I just noticed, that the release got stuck on Sonatype. Sorry.
There is a new version 1.1.64 released just now and is available in Maven Central.
